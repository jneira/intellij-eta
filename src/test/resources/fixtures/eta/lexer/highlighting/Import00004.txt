ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Import00004')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITimport ('import')
WHITE_SPACE (' ')
ITconid ('Data.QuickCheck')
WHITE_SPACE (' ')
IToparen ('(')
IToparen ('(')
ITvarsym ('===>')
ITcparen (')')
ITcparen (')')
WHITE_SPACE ('\n')
