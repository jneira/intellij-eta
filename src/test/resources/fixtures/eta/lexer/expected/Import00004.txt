ITmodule ('module')
WHITE_SPACE ('')
ITconid (' Import0000')
WHITE_SPACE ('4')
ITwhere (' wher')
WHITE_SPACE ('e\n\n')
ITvocurly ('')
WHITE_SPACE ('')
ITimport ('import')
WHITE_SPACE (' ')
ITqconid ('Data.QuickCheck')
WHITE_SPACE (' ')
IToparen ('(')
WHITE_SPACE ('')
IToparen ('(')
WHITE_SPACE ('')
ITvarsym ('===>')
WHITE_SPACE ('')
ITcparen (')')
WHITE_SPACE ('')
ITcparen (')')
WHITE_SPACE ('\n')
ITsemi ('')